docs/spec/errors.md

```diff
@@ -119,37 +119,39 @@ GET /api/serving.knative.dev/v1alpha1/namespaces/default/configurations/my-servi
 ```
 
 ```yaml
-...
+
+---
```

Comment:
We should remove the "..." rather than replacing it with the fragment separator.
```suggestion
```